eng/Subsets.props

```diff
@@ -113,6 +113,7 @@
   <PropertyGroup>
     <!-- CLR NativeAot only builds in a subset of the matrix -->
     <NativeAotSupported Condition="('$(TargetOS)' == 'windows' or '$(TargetOS)' == 'linux' or '$(TargetOS)' == 'osx' or '$(TargetOS)' == 'maccatalyst' or '$(TargetOS)' == 'iossimulator' or '$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvossimulator' or '$(TargetOS)' == 'tvos' or '$(TargetOS)' == 'freebsd') and ('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'arm64')">true</NativeAotSupported>
+    <NativeAotSupported Condition="'$(TargetOS)' == 'linux' and '$(TargetArchitecture)' == 'arm'">true</NativeAotSupported>
```

Comment:
Maybe we can just be add arm to the `('$(TargetArchitecture)' == 'x64' or '$(TargetArchitecture)' == 'arm64')` above? Windows ARM32 support was deleted in CoreCLR and ARM32 on osx or ioslike is also not relevant so this is unlikely to add something harmful.

Reply:
Makes sense. I'll update it.